server/middlewares/createPostValidator.js

```diff
@@ -0,0 +1,47 @@
+import validator from 'validator';
+
+const createPostValidation = async (req, res, next) => {
+  const { title, body } = req.body;
+
+  try {
```

Comment:
@Chiazokam there is a try-catch wrapper already made, avoid creating more try catch, it would defeat the purpose of the wrapper